src/clr-angular/data/tree-view/tree-node.html

```diff
@@ -6,7 +6,7 @@
 
 <div class="clr-tree-node-content-container">
   <button
-    *ngIf="isExpandable()"
+    *ngIf="isExpandable() && !_model.loading && !expandService.loading"
```

Comment:
To mirror how the `clr-tree-spinner-container` is displayed, should this be `isExpandable() && (!_model.loading || !expandService.loading)`?

Reply:
It would have to be `isExpandable() && !(_model.loading || expandService.loading)`. If you feel it's more readable, I can do that, but I tend to avoid mixing `&&` and `||` in boolean conditions for clarity.

Reply:
Right I missed the `!`.
I don't feel strong about how it is now. Thats why I already approved it.